rest/config.go

```diff
@@ -72,7 +72,7 @@ type BucketConfig struct {
 
 func (bc *BucketConfig) MakeBucketSpec() base.BucketSpec {
 
-	server := "http://localhost:8091"
+	server := "couchbases://localhost:8091"
```

Comment:
8091 is a http port, need to remove to allow it to default to 11207 ([memcached_ssl_port](https://docs.couchbase.com/server/current/install/install-ports.html#detailed-port-description))

```suggestion
	server := "couchbases://localhost"
```